tools/wpt/browser.py

```diff
@@ -464,29 +464,32 @@ def find_webdriver(self, channel=None):
     def _latest_chromedriver_url(self, browser_binary=None):
         latest = None
         chrome_version = self.version(browser_binary)
-        if chrome_version is not None:
-            parts = chrome_version.split(".")
-            if len(parts) == 4:
-                latest_url = "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_%s.%s.%s" % (
-                    parts[0], parts[1], parts[2])
+        assert chrome_version, "Cannot detect the version of Chrome"
+
+        # Remove channel suffixes (e.g. " dev").
+        chrome_version = chrome_version.split(' ')[0]
+        parts = chrome_version.split(".")
+        if len(parts) == 4:
+            latest_url = "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_%s.%s.%s" % (
+                parts[0], parts[1], parts[2])
```

Comment:
Let's just make this `% parts[0:3]`

Reply:
Actually, `%` doesn't like lists. I have to convert it to a tuple.